doc-content/enterprise-only/openshift/rhpam72-prod-immutable-monitor-ref.adoc

```diff
@@ -32,11 +32,17 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`KIE_SERVER_USER` | `KIE_SERVER_USER` | KIE server username (Sets the org.kie.server.user system property) | executionUser | False
 |`KIE_SERVER_PWD` | `KIE_SERVER_PWD` | KIE server password, used to connect to KIE servers. Generated value can be a suggestion to use for thew s2i various (Sets the org.kie.server.pwd system property) | `${KIE_SERVER_PWD}` | False
 |`IMAGE_STREAM_NAMESPACE` | -- | Namespace in which the ImageStreams for Red Hat Middleware images are installed. These ImageStreams are normally installed in the openshift namespace. You should only need to modify this if you installed the ImageStreams in a different namespace/project. | openshift | True
-|`IMAGE_STREAM_TAG` | -- | A named pointer to an image in an image stream. Default is "1.0". | 1.0 | False
+|`IMAGE_STREAM_TAG` | -- | A named pointer to an image in an image stream. Default is "1.1". | 1.1 | False
+|`SMART_ROUTER_HOSTNAME_HTTP` | `HOSTNAME_HTTP` | Custom hostname for http service route.  Leave blank for default hostname, e.g.: <application-name>-smartrouter-<project>.<default-domain-suffix> | `${BUSINESS_CENTRAL_HOSTNAME_HTTP}` | False
```

Comment:
${BUSINESS_CENTRAL_HOSTNAME_HTTP} for the example value should be ${SMART_ROUTER_HOSTNAME_HTTP}

Reply:
This is content autogenerated from the template. I have removed the environment variable (as in previous line) but this means the docs are generated wrong or the template is wrong. Can we raise this further?